### PR TITLE
More announcement tweaks 🔈

### PR DESCRIPTION
**Pull request by FlattestGuitar**
My last PR missed a bit of code, so I figured I might as well change this whole bit here.

Admins have access to the different types of announcements way faster now, and the whole thing is a bit cleaner, though the end result is still pretty much the same.

No CL because it's backstage.

### CODE REVIEW

**Comment by Kyep** [comment]
Consider:
- Having a "Classified Message" type.
- Having an "Orders from Central Command" type.
- Allowing us to specify the sound played with the announcement.

**Comment by AndriiYukhymchak** [comment]
Having "Classified Message" type will be cool to throw off players thinking about blob round

**Comment by FlattestGuitar** [comment]
You can now pick from a few sounds, and any report that you choose not to show to the whole station will produce the same message as the blob one.

File: code/defines/procs/announce.dm [hunk 1]
```diff
@@ -137,20 +129,10 @@
...
-	var/formatted_message
-	formatted_message += "<h1 class='alert'>[title]</h1>"
-	if(message_title)
-		formatted_message += "<br><h2 class='alert'>[message_title]</h2>"
-	formatted_message += "<br><span class='alert'>[message]</span><br>"
-	formatted_message += "<br>"
-	
 	return formatted_message
 
-/datum/announcement/priority/enemy/Format_Message(message, message_title, message_announcer, from)
```
Comment (FlattestGuitar) [inline code/defines/procs/announce.dm]: All admin announcements now use `/datum/announcement/priority/command`, with changing titles, sounds and all that, so there's not much point in keeping the enemy announcement thingy here.

**Pull request merged**